src/skmultiflow/drift_detection/hddm_w.py

```diff
@@ -131,16 +139,21 @@ def add_element(self, prediction):
     def _detect_mean_increment(self, sample1, sample2, confidence):
         if sample1.EWMA_estimator < 0 or sample2.EWMA_estimator < 0:
             return False
-
-        bound = sqrt((sample1.independent_bounded_condition_sum
-                      + sample2.independent_bounded_condition_sum) * log(1 / confidence) / 2)
+        sum = sample1.independent_bounded_condition_sum + sample2.independent_bounded_condition_sum
```

Comment:
`sum` is a reserved keyword in Python. I suggest changing the variable name. Appending a simple underscore after the word is enough: `sum_` (though I am not sure it is the best practice/workaround).

Reply:
Nice catch!

Reply:
:D